zigzag/cost_model/cost_model.py

```diff
@@ -689,7 +689,7 @@ def calc_double_buffer_flag(self) -> None:
 
         self.double_buffer_true = double_buffer_true
 
-    def __calc_allowed_and_real_data_transfer_cycle_per_DTL(self):
```

Comment:
What is the reasoning behind making the private functions public again?

Reply:
These functions need to be called within cost_model_imc when inheriting this class. So, it's better keep them to be public.

Reply:
Oh right, I forgot `CostModelEvaluationIMC` inherits from `CostModelEvaluation`. Sorry!